prover/src/tests/training_wheels.rs

```diff
@@ -45,7 +47,36 @@ fn test_validate_jwt_sig_and_dates_expired() {
     assert!(validate_jwt_sig_and_dates(
         &prover_request_input,
         Some(&jwk_keypair.into_rsa_jwk()),
-        false
+        &CONFIG,
+    )
+    .is_ok());
+}
+
+#[test]
+fn test_validate_jwt_sig_and_dates_expired_can_be_disabled() {
```

Comment:
This test differs from the one above only in config change, right? (i.e., `    config.enable_jwt_exp_not_in_the_past_check = false;`)

Why not avoid re-writing the whole test cases by using a helper function that takes the config as a parameter?

Reply:
I think the `iat` test may have the same issue?